text/3-entrees/basic-pow.md

```diff
@@ -114,7 +114,7 @@ pub fn build_inherent_data_providers() -> Result<InherentDataProviders, ServiceE
 Anything that implements the
 [`ProvideInherentData` trait](https://crates.parity.io/sp_inherents/trait.ProvideInherentData.html)
 may be used here. The block authoring logic must supply all inherents that the runtime expects. In
-this case of this basic-pow chain, that is just the
+the case of this basic-pow chain, that is just the
```

Comment:
I would suggest using either `basic-pow` or "basic POW", depending on the context.

Reply:
Agree. We have an issue for it https://github.com/substrate-developer-hub/recipes/issues/273. Don't want to create too many opportunities for merge conflicts here.